packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -31,13 +32,16 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
       'Some details about the call that span more than one line - many, many lines in fact. Who would want fewer lines than many, many lines? Could you even imagine?! 😲';
   }
 
+  const ParticipantItemOptions = queryArgs.showParticipantItemIcon ? <MoreHorizontal20Regular /> : <></>;
```

Comment:
wow nice love to see tests for obscure, easily-forgetten things!

Reply:
Should these new snapshots have the participant menu showing?
![image](https://user-images.githubusercontent.com/2684369/190526892-8c72b366-9d60-49b6-a9b0-1b50eb396ecb.png)

Reply:
yes they should! let me update this